exercicios/para-casa/anna-berto/choiceDrink.js

```diff
@@ -0,0 +1,53 @@
+import drinks from "./drinks";
+
+function listMenuDrink() {
+  const listMenu = drinks.map((drink) => {
+    return {
+      name: drinks.name,
+    };
+  });
+  console.log(listMenu);
+}
+
+function drinkDetails() {
+  const listDrinkDetails = drinks.map((drinks) => {
+    return {
+      name: drinks.name,
+      price: drinks.price,
+      brand: drinks.brand,
+      typedrink: drinks.typeDrink,
+      flavor: drinks.flavor,
+    };
+  });
+  console.log(listDrinkDetails);
+  console.log("Press 0 to return.");
+
+  var input = document.querySelector("#choice");
+  var choice = input.value;
+  switch (choice) {
+    case "0":
+      listMenuDrink();
+  }
+}
+
+choceDrink = () => {
+  var input = document.querySelector("#choice");
+  var choice = input.value;
+
+  switch (choice) {
```

Comment:
<p>Nessa parte está bem organizada e com a lógica simples de entender  mas se me permitir deixar uma sugestão de melhoria, eu acredito que nesse caso não precisaria do switch, porque todos os cases executam o mesmo código.</p>

---
Pensando em reaproveitamento de código, acredito que um if verificando se `choice` é um número e se está entre 1 a 6 resolveria, porque nesse caso, só precisaria chamar o método `drinkDetails()` uma vez e a opção do default poderia virar um else com um `throw new Error("We couldn't find this drink, try again")`.